thirdparty/jansson/value.c

```diff
@@ -756,6 +760,68 @@ static json_t *json_string_copy(json_t *string)
     return json_string_nocheck(json_string_value(string));
 }
 
+// Given a json array, concat the elements using a delimiter
+w_string_t *w_string_implode(json_t *arr, const char *delim)
```

Comment:
Seems like this functions isn't used anywhere. I'll get rid of it altogether (as discussed offline with @sid0).